net/haproxy/src/etc/rc.syshook.d/50-haproxy.restart

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
this seems like a misconception, we only support: early (early system boot before network setup), start (later system boot when all services are started after the core is set up), stop (during shutdown or reboot where services are stopped)